phuzzer/phuzzers/afl_ijon.py

```diff
@@ -0,0 +1,19 @@
+from .afl import AFL
```

Comment:
Ideally for new files and when updating imports, we should try to sort them into three sections: standard lib, third-party, and local, alphabetically sorted within each section.

Reply:
fixed